### ISSUE TITLE
Ability to submit bugs

### ISSUE THREAD

**Issue by chinyeungli** [nexB/scancode.io#122]
Ideally, when viewing a file and spot bugs, user can select the texts and submit bugs.

One approach is to auto-generated a github ticket (while the user still needs to use his/her own github ID)

Other possible/potential approaches are welcome to be discussed.

**Comment by AyanSinhaMahapatra** [comment]
Using the [analyzer](https://github.com/nexB/scancode-results-analyzer) to spot issues.

Results file after applying the analyzer post-scan plugin on the Debian copyright files: [copyrights-debian-analyzer.json.txt](https://github.com/nexB/scancode.io/files/6238625/copyrights-debian-analyzer.json.txt)

1. Spots the license detection issues and classifies them 
2. Groups the license matches in a location together into one issue, with a combined text. 
3. If the same issues appear elsewhere in the scan list them together and show the issue once.

IMHO this could help with the issue review and bug submission process a lot.

Though note that, license-expression suggestions are only preliminary and based on the faulty matches themselves, so they work in only straightforward cases.

**Comment by tdruez** [comment]
@AyanSinhaMahapatra could you provide all the steps (with code) on how you run and apply the analyzer?
The link @ https://github.com/nexB/scancode-results-analyzer#documentation is not working by the way.
Also, any reasons for not publishing this tool on pypi? We cannot integrate it in ScanCode.io until then...

**Comment by AyanSinhaMahapatra** [comment]
@tdruez Yes, I had requested for the RTD site to be added in slack, as I do not have access to add repos. I do have it set up for my fork - https://scancode-results-analyzer-fork-ayan.readthedocs.io/

Installation and running the plugin works as described here - https://github.com/nexB/scancode-results-analyzer/blob/main/INSTALL.rst

No there's no reasons for not publishing on pypi, but @pombredanne and I were still discussing on the details and report format, fixing some issues, so it got delayed. We are working on that, to then integrate with scancode.io 

Related tickets are -> 

- https://github.com/nexB/scancode-results-analyzer/issues/21
- https://github.com/nexB/scancode.io/issues/31

**Comment by AyanSinhaMahapatra** [comment]
Btw, nexB/scancode-analyzer#21 is closed now, `scancode-analyzer` is available as a [pypi package](https://pypi.org/project/scancode-analyzer/) and docs is at https://scancode-analyzer.readthedocs.io/en/latest/

**Comment by tdruez** [comment]
@AyanSinhaMahapatra that's great news. I'll start a branch implementing `scancode-analyzer` as a pipeline (we have several pending tickets on this).

**Comment by AyanSinhaMahapatra** [comment]
So this ticket on designing and having a format to review/resolve issues in scancode.io #137 would be needed before that? I can start with integrating analyzer in a pipeline then with your help?

**Comment by AyanSinhaMahapatra** [comment]
@pombredanne -> https://github.com/sindresorhus/new-github-issue-url/issues/new?title=adadasd&body=%0A%0A%0A---%0AI'm+a+human.+Please+be+nice. 

Example of using URL formatting to create new issues.

**Comment by pombredanne** [comment]
See also https://github.com/foss-dev/new-github-issue-url-py
And https://docs.github.com/en/github/managing-your-work-on-github/about-automation-for-issues-and-pull-requests-with-query-parameters

**Comment by pombredanne** [comment]
Or https://github.com/sindresorhus/new-github-issue-url in JS
